src/frontend/src/handler/alter_set_schema.rs

```diff
@@ -189,49 +189,3 @@ pub async fn handle_alter_set_schema(
 
     Ok(RwPgResponse::empty_result(stmt_type))
 }
-
-#[cfg(test)]
-pub mod tests {
-    use risingwave_common::catalog::{DEFAULT_DATABASE_NAME, DEFAULT_SCHEMA_NAME};
-
-    use crate::catalog::root_catalog::SchemaPath;
-    use crate::test_utils::LocalFrontend;
-
-    #[tokio::test]
-    async fn test_alter_set_schema_handler() {
-        let frontend = LocalFrontend::new(Default::default()).await;
-        let session = frontend.session_ref();
-
-        let sql = "CREATE TABLE test_table (u INT, v INT);";
-        frontend.run_sql(sql).await.unwrap();
-
-        let sql = "CREATE SCHEMA test_schema;";
-        frontend.run_sql(sql).await.unwrap();
-
-        let get_table = |schema_name| {
-            let catalog_reader = session.env().catalog_reader().read_guard();
-            let schema_path = SchemaPath::Name(schema_name);
-            catalog_reader
-                .get_created_table_by_name(DEFAULT_DATABASE_NAME, schema_path, "test_table")
-                .unwrap()
-                .0
-                .clone()
-        };
-
-        let get_schema_name_by_table_id = |table_id| {
-            let catalog_reader = session.env().catalog_reader().read_guard();
-            catalog_reader
-                .get_schema_by_table_id(DEFAULT_DATABASE_NAME, &table_id)
```

Comment:
The (test-only) `get_schema_by_table_id` looks strange. We already have e2e test `e2e_test/ddl/alter_set_schema.slt`, I think it's fine to just remove UT.